pelican/log.py

```diff
@@ -194,11 +179,13 @@ def get_formatter():
         return TextFormatter()
 
 
+class FatalHandler(logging.Handler):
+    def emit(self, record):
+        sys.exit(1)
```

Comment:
Very minor [suggestion](https://adamj.eu/tech/2021/10/10/the-many-ways-to-exit-in-python/), but perhaps change `sys.exit(1)` to the following? 

```python
raise SystemExit(1)
```